### PR TITLE
update releasing.md

### PR DESCRIPTION
**Pull request by tuminoid**
With the learnings of making latest minor release, update docs.

### CODE REVIEW

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

File: processes/releasing.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ be built automatically  with `capm3-v1.x.y` tag in Quay. Verify the build is
 triggered, as often Quay has disabled the build trigger due build failures or
 other reasons.
 
+## Update Metal3 dev-env and CI to support new releases
+
+If a new minor or major release was published, we need to make changes in
+Metal3 dev-env and CI.
+**NOTE**: These changes shall only be made after the releases are out, as they
```
Comment (kashifest) [inline processes/releasing.md:45]:
```suggestion
**NOTE**: These changes shall only be merged after the releases are out, as they
```

**Review by kashifest — COMMENTED** [review]
/approve
A small nit

File: processes/releasing.md (hunk 1 above)
Comment (tuminoid) [reply processes/releasing.md:45]: thanks, fixed.

**Pull request merged**